packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "3.53.0",
+  "version": "3.53.1-fb-multiline-grid.2",
```

Comment:
Note: Inability to copy multi-line content from a display cell to another display cell is a pre-existing non-feature/bug. I looked into what sheets does for this and it looks like wrapping in quotes is the way they identify multi-line text from the clipboard.

Reply:
I tried this out and it is definitely a little surprising behavior to the user when you copy a single cell and then paste it to have the data split into multiple rows, but I get it. Copy and paste across cells for multi line data seems likely an unlikely use-case, so seems fine to me to defer this. Note that the fill down behavior does work as expected here.